rdkit/Chem/FeatMaps/FeatMapParser.py

```diff
@@ -162,30 +160,19 @@ def ParseFeatPointBlock(self):
       p = FeatMapPoint.FeatMapPoint()
 
       i = 0
-      while i < len(vals):
+      for i in range(0, len(vals), 2):
```

Comment:
That works too. I make only one additional suggestion:
```
      for i in range(0, len(vals), 2):
        name = vals[i].lower()
        value = vals[i + 1]
        if name == 'family':
          p.SetFamily(value.strip())
        elif name == 'weight':
          p.weight = float(value)
        elif name == 'pos':
          p.SetPos(self._parsePoint(value))
        elif name == 'dir':
          p.featDirs.append(self._parsePoint(value))
        else:
          raise FeatMapParseError(f'FeatPoint option {name} not recognized on line {self._lineNum}')
```
This makes it immediately clear that we parse the list as pairs (name, value)

Reply:
OK, I got your idea! I have fixed it. Thank you for your review.